src/main/java/nl/hsac/fitnesse/fixture/slim/HttpTest.java

```diff
@@ -749,6 +750,12 @@ public void setContentType(String aContentType) {
         contentType = aContentType;
     }
 
+    public void setBasicAuthorizationHeaderForUserAndPassword(String user, String password) {
+        String credential = user + ":" + password;
+        String base64credential = Base64.getEncoder().encodeToString(credential.getBytes());
```

Comment:
Please add an explicit encoding (using 'ISO-8859-1') when converting the string to bytes.